### PR TITLE
Add Solidity Survey to Feature Section

### PR DESCRIPTION
**Pull request by joeizang**
Adds feature section to featured section carousel on hometab to remind developers about solidity developer survey.

### CODE REVIEW

File: libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx [hunk 1]
```diff
@@ -35,17 +36,28 @@ function HomeTabFeatured() {
...
+                  <p style={{ fontStyle: 'italic' }}><FormattedMessage id='home.solidityDevSurvey1' /></p>
+                  <p>
+                    <FormattedMessage id='home.solidityDevSurvey' />
+                  </p>
+                  <a className="remixui_home_text" target="__blank" onClick={() => _paq.push(['trackEvent', 'hometab', 'featuredSection', 'soliditySurvey'])} href="https://blog.soliditylang.org/2022/12/07/solidity-developer-survey-2022-announcement/"><FormattedMessage id='home.surveyLink' /></a>
+                </div>
+              </div>
               <div className="mx-1 px-1 d-flex">
                 <img src={"assets/img/bgRemi.webp"} style={{ flex: "1", height: "170px", maxWidth: "170px" }} alt="" ></img>
-                <div className="h6 w-50 p-4" style={{ flex: "1" }}>
```
Comment (LianaHus) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:40]: why?

Comment (joeizang) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:40]: my mistake

**Review by LianaHus — CHANGES_REQUESTED** [review]
the color of the logo should b e updated for themes and it needs some space around ( should be smaller)

File: libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx [hunk 2]
```diff
@@ -45,18 +56,18 @@ function HomeTabFeatured() {
...
                   <a className="remixui_home_text" target="__blank" onClick={() => _paq.push(['trackEvent', 'hometab', 'featuredSection', 'remixRewards'])} href="https://rewards.remix.ethereum.eth.limo"><FormattedMessage id='home.more' /></a>
                 </div>
               </div>
               <div className="mx-1 px-1 d-flex">
                 <img src={"/assets/img/remixRewardBetaTester.webp"} style={{ flex: "1", height: "170px", maxWidth: "170px" }} alt="" ></img>
-                <div className="h6 p-4" style={{ flex: "1" }}>
+                <div className="h6 w-50 p-4" style={{ flex: "1" }}>
                   <h5><FormattedMessage id='home.betaTesting' /></h5>
-                  <p style={{ fontStyle: 'italic' }}><FormattedMessage id='home.betaTestingText1' /></p>
+                  {/* <p style={{ fontStyle: 'italic' }}><FormattedMessage id='home.betaTestingText1' /></p> */}
```
Comment (LianaHus) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:70]: why are you removing those?

Comment (joeizang) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:70]: @Aniket-Engg removed those.

Comment (Aniket-Engg) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:70]: links are not visible if we keep them. Also, they are not in `Jump into web3` slide. We can bring them back once UI is fixed/redesigned.

Comment (LianaHus) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:70]: change the font size or spaceing

Comment (LianaHus) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:70]: or move links to the same line

File: libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx [hunk 3]
```diff
@@ -35,6 +36,16 @@ function HomeTabFeatured() {
...
+                  <h5><FormattedMessage id='home.solidityDevSurveyHeader' /></h5>
+                  <p className="remixui_home_carouselText">
+                    <FormattedMessage id='home.solidityDevSurvey' />
+                  </p>
+                  <a className="remixui_home_text btn btn-primary text-decoration-none mb-3" target="__blank" onClick={() => _paq.push(['trackEvent', 'hometab', 'featuredSection', 'soliditySurvey'])} href="https://blog.soliditylang.org/2022/12/07/solidity-developer-survey-2022-announcement/"><FormattedMessage id='home.surveyLink' /></a>
+                </div>
+              </div>
               <div className="mx-1 px-1 d-flex">
                 <img src={"assets/img/bgRemi.webp"} style={{ flex: "1", height: "170px", maxWidth: "170px" }} alt="" ></img>
                 <div className="h6 w-50 p-4" style={{ flex: "1" }}>
```
Comment (LianaHus) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:51]: you can out p-2 instead to have more space

File: libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx [hunk 4]
```diff
@@ -32,33 +33,45 @@ function HomeTabFeatured() {
               sliderClass="h-100 justify-content-between"
               deviceType={"desktop"}
               itemClass=""
-              autoPlaySpeed={15000}
+              autoPlaySpeed={150000}
```
Comment (LianaHus) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:36]: remember to change it back before merging this :)

Comment (joeizang) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:36]: oh yes. taking it back now

File: apps/remix-ide/src/app/tabs/locales/zh/home.json [hunk 5]
```diff
@@ -12,6 +12,10 @@
   "home.remixRewards": "REMIX 奖励",
   "home.remixRewardsText1": "给我们的用户提供的 NFT!",
   "home.remixRewardsText2": "Remix 会用部署在 Optimism 链上的 NFT 奖励贡献者、beta 测试者和用户体验研究参与者。Remix 奖励的持有者可以铸造第二个“Remixer”用户 NFT 徽章，以赠予他们选择的任何其他用户。",
+  "home.solidityDevSurveyHeader": "",
```
Comment (yann300) [inline apps/remix-ide/src/app/tabs/locales/zh/home.json:15]: please remove the keys

File: apps/remix-ide/src/app/tabs/locales/zh/home.json [hunk 6]
```diff
@@ -12,6 +12,10 @@
   "home.remixRewards": "REMIX 奖励",
   "home.remixRewardsText1": "给我们的用户提供的 NFT!",
   "home.remixRewardsText2": "Remix 会用部署在 Optimism 链上的 NFT 奖励贡献者、beta 测试者和用户体验研究参与者。Remix 奖励的持有者可以铸造第二个“Remixer”用户 NFT 徽章，以赠予他们选择的任何其他用户。",
```
Comment (drafish) [inline apps/remix-ide/src/app/tabs/locales/zh/home.json:14]: Remix 会用部署在 Optimism 链上的 NFT 奖励贡献者、beta 测试者和用户体验研究参与者。

File: apps/remix-ide/src/app/tabs/locales/zh/home.json (hunk 5 above)
Comment (drafish) [inline apps/remix-ide/src/app/tabs/locales/zh/home.json:15]: SOLIDITY 开发者 2022 年度调查问卷

File: apps/remix-ide/src/app/tabs/locales/zh/home.json [hunk 7]
```diff
@@ -12,6 +12,10 @@
...
+  "home.solidityDevSurveyHeader": "",
+  "home.solidityDevSurvey": "",
```
Comment (drafish) [inline apps/remix-ide/src/app/tabs/locales/zh/home.json:16]: Solidity 团队正在收集反馈以改进 Solidity 。调查问卷大约需要 10 分钟。

File: apps/remix-ide/src/app/tabs/locales/zh/home.json [hunk 8]
```diff
@@ -12,6 +12,10 @@
...
+  "home.solidityDevSurvey1": "",
```
Comment (drafish) [inline apps/remix-ide/src/app/tabs/locales/zh/home.json:17]: 年度投票

**Review by LianaHus — APPROVED** [review]

**Pull request merged**